src/Isolation/Masa.Contrib.Isolation.UoW.EF/Parser/MultiTenancy/HeaderTenantParserProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Masa.Contrib.Isolation.UoW.EF.Parser.MultiTenancy;
+
+public class HeaderTenantParserProvider : ITenantParserProvider
+{
+    public string Name => "Header";
```

Comment:
How to use for each Parser?

Reply:
Need to explain how each parser is used here?

Reply:
doc

Reply:
ok, the parser here is very simple, so I did not specifically explain how to use the parser, I will add it later